src/stan/mcmc/windowed_adaptation.hpp

```diff
@@ -112,14 +107,14 @@ class windowed_adaptation : public base_adaptation {
  protected:
   std::string estimator_name_;
 
-  unsigned int num_warmup_;
-  unsigned int adapt_init_buffer_;
-  unsigned int adapt_term_buffer_;
-  unsigned int adapt_base_window_;
+  unsigned int num_warmup_{0};
+  unsigned int adapt_init_buffer_{0};
+  unsigned int adapt_term_buffer_{0};
+  unsigned int adapt_base_window_{0};
 
-  unsigned int adapt_window_counter_;
-  unsigned int adapt_next_window_;
-  unsigned int adapt_window_size_;
+  int adapt_window_counter_{-1};
+  unsigned int adapt_next_window_{0};
```

Comment:
This is what you were worried about overflowing, but it is still an `unsigned int`.

Reply:
Gaaah dumb on my part